lib/alchemy/upgrader/tasks/cells_migration.rb

```diff
@@ -35,5 +37,23 @@ def migrate_cell!(cell)
         puts "Element for cell '#{cell.name}' already present. Skip"
       end
     end
+
+    def remove_cell_reference_from_elements_table
+      puts "-- Removing cell_id from alchemy_elements table."
+      migration = ActiveRecord::Migration[5.2].new
+      migration.remove_reference :alchemy_elements, :cell, index: true, foreign_key: {
+        to_table: :alchemy_cells
+      }
+    rescue ArgumentError
+      puts "-> No cell_id found on alchemy_elements table. Skip"
+    end
+
+    def drop_cells_table
+      puts "-- Dropping cells table."
+      migration = ActiveRecord::Migration[5.2].new
```

Comment:
No, it won't. This can't be an extra migration as migrations run before the upgrader run. And this can't be change as most upgrades expect the schema to be in a certain state. I know that this is not ideal, but easier than "fixing" the upgrader. The other option we have is to not drop the table and column at all. Won't hurt anyone and who ever is annoyed by unused tables and columns can remove them if they like.

Maybe this the best option, actually

Reply:
Done ef845f7a9f9106bfcb33017783caa69a9eb93bc8 and c6ad8981051574ec16ba4623b36f533990323921